Assets/Plugins/iOS/Core/EOS_iOSLoginOptionsHelper.cs

```diff
@@ -0,0 +1,41 @@
+using System;
```

Comment:
This needs to have a license added to the top of the file.

Reply:
Ready, does the I added the license with year 2023, is that correct